integration-tests/slack/circleci-utils.js

```diff
@@ -0,0 +1,66 @@
+const _ = require('lodash/fp')
+const fetch = require('node-fetch')
+const { parse } = require('path')
+
+
+/**
+ * Fetch CircleCI artifact links to tests summary JSON files (created in onRunComplete() in jest-reporter.js)
+ * @param {string} token
+ * @param {string} buildNum
+ * @returns {Promise<Array[string]>} URL to tests-summary-[0-9].json
+ */
+const fetchJobArtifacts = async ({ buildNum = process.env.CIRCLE_BUILD_NUM } = {}) => {
+  if (!buildNum) {
+    throw new Error(`**  ERROR: Missing CircleCI build number. Failed to fetch CircleCI job artifacts.`)
+  }
+
+  // For more arguments and details of the response, see: https://circleci.com/docs/api/v2/index.html#operation/getJobArtifacts
+  const apiUrlRoot = 'https://circleci.com/api/v2/project/github/DataBiosphere/terra-ui'
+  try {
+    // Because terra-ui is a public repository on GitHub, API token is not required. See: https://circleci.com/docs/oss#security
+    const response = await fetch(`${apiUrlRoot}/${buildNum}/artifacts`)
+    const { items } = await response.json()
+    const testSummaryArtifacts = _.filter(_.flow(_.get('path'), _.includes('tests-summary')), items)
+    return _.map(_.get('url'), testSummaryArtifacts)
```

Comment:
Nit: FYI, [`_.map`](https://lodash.com/docs/4.17.15#map) supports a `_.property` shorthand, so you can simply do:
```suggestion
    return _.map('url', testSummaryArtifacts)
```
(Note: this is from memory and reading the docs; I didn't actually test this suggestion)

Reply:
I had no idea. Suggested code works!